ui/src/testing/stubs/cohort-annotation-definition-service-stub.ts

```diff
@@ -19,7 +19,7 @@ export class CohortAnnotationDefinitionServiceStub extends CohortAnnotationDefin
   }
 
   deleteCohortAnnotationDefinition(): Promise<EmptyResponse> {
-    return new Promise<EmptyResponse>(resolve => resolve());
+    return new Promise<EmptyResponse>(resolve => resolve(null));
```

Comment:
Resolved `error TS2794: Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?`